File: support_tools/swigwin-3.0.12/Examples/test-suite/go/return_const_value_runme.go
package main

import "./return_const_value"

func main() {
	p := return_const_value.Foo_ptrGetPtr()
	if p.GetVal() != 17 {
		panic("Runtime test1 failed")
	}

	p = return_const_value.Foo_ptrGetConstPtr()
	if p.GetVal() != 17 {
		panic("Runtime test2 failed")
	}
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/return_const_value_runme.py
import return_const_value
import sys

p = return_const_value.Foo_ptr_getPtr()
if (p.getVal() != 17):
    print "Runtime test1 faild. p.getVal()=", p.getVal()
    sys.exit(1)

p = return_const_value.Foo_ptr_getConstPtr()
if (p.getVal() != 17):
    print "Runtime test2 faild. p.getVal()=", p.getVal()
    sys.exit(1)
